commercetools-models/src/test/java/io/sphere/sdk/carts/ExternalTaxRatesIntegrationTest.java

```diff
@@ -304,7 +304,7 @@ public void setShippingMethodTaxRate() {
     @Test
     public void addLineItemOnPlatformCart() {
         withProductOfPrices(client(), singletonList(PriceDraft.of(EURO_10)), (Product product) -> {
-            final CartDraft draft = CartDraft.of(EUR);//uses by default platform!
+            final CartDraft draft = CartDraft.of(EUR);//uses by default Composable Commerce!
```

Comment:
I'm not quite sure what this means

Reply:
@jenschude any help here please?

Reply:
I'm reverting it to "Platform" as it means that the Platform is used to calculate the taxmode
https://docs.commercetools.com/api/projects/carts#taxmode

Reply:
Cool, thanks! Could you also update the phrasing to be a bit clearer, something like: 'Uses the Platform TaxMode by default'